src/ol/Feature.js

```diff
@@ -231,9 +236,9 @@ Feature.prototype.setGeometry = function(geometry) {
  * Set the style for the feature.  This can be a single style object, an array
  * of styles, or a function that takes a resolution and returns an array of
  * styles. If it is `null` the feature has no style (a `null` style).
- * @param {ol.style.Style|Array.<ol.style.Style>|ol.StyleFunction} style Style for this feature.
+ * @param {module:ol/style/Style~Geometry|Array.<module:ol/style/Style~Geometry>|module:ol/style~StyleFunction} style Style for this feature.
```

Comment:
@ahocevar - I'm guessing that this should be `module:ol/style/Style~Style`, right?

See #7961.

Reply:
Absolutely. Looks like that was too much search and replace.